site/_wiki/Install/Linux.md

```diff
@@ -24,7 +24,7 @@ If you have the Microsoft dotnet repository installed you will have to either ma
 
 If you're experiencing `libhostfxr` issues, please see the solutions from Microsoft [here]({{ site.data.links.external.Microsoft.DotnetLinuxPackageMixup }}).
 
-## Fedora {#rpm}
```

Comment:
Should probably make a new section here instead with a title like "RPM-based distributions" and then use H3's for Fedora and OpenSuSE respectively instead